src/apps/lwaftr/ctable_wrapper.lua

```diff
@@ -73,12 +73,12 @@ function selftest(params)
    for j=0,5 do v[j] = bnot(i) end
    local newest_index = ctab:add_with_random_ejection(i, v)
    local iterated = 0
-   for entry in ctab:iterate() do iterated = iterated + 1 end
+   for _ in ctab:iterate() do iterated = iterated + 1 end
    assert(old_occupancy == ctab.occupancy, "bad random ejection!")
  
    ctab:remove_ptr(ctab.entries + newest_index, false)
-   local iterated = 0
-   for entry in ctab:iterate() do iterated = iterated + 1 end
+   iterated = 0
+   for _ in ctab:iterate() do iterated = iterated + 1 end
```

Comment:
i think it's useful to have names when it's unclear what iteration produces.  since selftest code is often example code I think i would keep the names if it doesn't make things too verbose; dunno